libs/portals/shared-modules/delegations/src/components/access/AccessCard.tsx

```diff
@@ -239,22 +245,24 @@ export const AccessCard = ({
           flexDirection={['column', 'row']}
           width="full"
         >
-          {hasTags && (
-            <Box width="full">
-              <VisuallyHidden>{formatMessage(m.accessScopes)}</VisuallyHidden>
-              <Inline alignY="bottom" space={1}>
-                {tags.map((tag, index) => (
-                  <Tag
-                    disabled
-                    key={index}
-                    variant={tag.isExpired ? 'disabled' : 'blue'}
-                  >
-                    {tag.name}
-                  </Tag>
-                ))}
-              </Inline>
-            </Box>
-          )}
+          <div>
```

Comment:
We should avoid using empty divs, make the justifyContent conditional to handle the positioning instead.